develop/integrations.rst

```diff
@@ -1,6 +1,183 @@
+
+If the :ref:`enable_change_data_capture` GUC is enabled, Citus automatically
+handles the issues from group two, normalizing shard names and suppressing
+events from shard creation and movement. Issues in group one, however, are the
+client's responsibility.
+
+Logical replication of distributed tables to PostgreSQL tables
+--------------------------------------------------------------
+
+Postgres' CDC with logical decoding forms the basis of `logical
+replication
+<https://www.postgresql.org/docs/current/logical-replication.html>`_, which
+allows you to replicate table changes from one PostgreSQL server to another.
+As an example, let's subscribe to changes from a distributed table in a Citus
+cluster, and populate a regular table in a single-node PostgreSQL server. 
+
+First, enable logical decoding on all Citus nodes. In the postgresql.conf file
+of each of them, configure the WAL and replication slots:
+
+.. code-block:: ini
+
+  # add to postgresql.conf on all nodes
+
+  # include enough information in WAL for logical decoding
+
+  wal_level = logical
+
+  # we need at least one replication slot, but leave headroom
+
+  max_replication_slots = 10
+
+From the coordinator node, enable Citus' logical replication improvements
+for distributed tables:
+
+.. code-block:: postgresql
+
+  -- run this on the coordinator node
+
+  SELECT * FROM run_command_on_all_nodes($$
+    ALTER SYSTEM SET citus.enable_change_data_capture TO 'on';
+  $$);
+
+After changing the configuration files, restart the PostgreSQL server in each
+Citus node.
+
+Create an example distributed table on the coordinator, and a publication to
+share changes from the table.
+
+.. code-block:: postgresql
+
+  -- example distributed table
+
+  CREATE TABLE items (
+    key TEXT PRIMARY KEY,
+    value TEXT
+  );
+
+  SELECT create_distributed_table('items', 'key');
+
+  -- citus will propagate to workers, creating publications there too
+
+  CREATE PUBLICATION items_pub FOR TABLE items;
+
+  -- fill with sample data
+
+  INSERT INTO items SELECT generate_series(1,10), 'a value';
+
+Check that the worker nodes have the correct hostname set for the coordinator,
+and adjust with :ref:`set_coordinator_host` if necessary. Then, create logical
+replication slots on all nodes by running the following on the coordinator:
+
+.. code-block:: postgresql
+
+  -- create replication slots on all nodes, and choose the pgoutput format
+
+  SELECT * FROM run_command_on_all_nodes($$
+    SELECT pg_create_logical_replication_slot('cdc_slot', 'pgoutput', false)
+  $$);
+
+Finally, on the single node PostgreSQL server, create the ``items`` table, and
+set up subscriptions to read rows from all Citus nodes.
+
+.. code-block:: postgresql
+
+  -- on the standalone PostgreSQL server
+
+  -- match the table structure we have on Citus
+
+  CREATE TABLE items (
+    key TEXT PRIMARY KEY,
+    value TEXT
+  );
+
+  -- subscribe to coordinator
+
+  CREATE SUBSCRIPTION subc
+    CONNECTION 'host=c.myclyster … '
```

Comment:
can we include the full CONNECTION string here?
users may just copy paste this as is..